storybookjs/storybook
# Serve styles and assets using .angular-cli webpack configuration

Issue: See #2645, #2688 

## What I did

Add ability to configure webpack using `@angular/cli` and user's `.angular-cli.json`.

So `@storybook/anguar` serves according to the .angular-cli.json: 

- Global styles
- Static asset files

And if user don't have .angular-cli.json, the minimum configuration is provided as before.

## How to test

See example/angular-cli manually 🙏 . I added new component and story under `examples/angular-cli/src/stories/component-with-style`. Open this story, you can see the following styled DOM:

- Scoped CSS(with templateURL)
- Scoped SCSS(with templateURL)
- Global CSS

And open `localhost:9008/assets/common.css`, you can confirm that static files listed .angular-cli.json are served.
